biz_portal/apps/portal/templates/portal/business_list.html

```diff
@@ -1,8 +1,89 @@
+{% extends "base.html" %}
 {% load param_replace %}
+{% load static %}
 
-{% include 'portal/search_snippet.html' %}
+<!-- {% include 'portal/search_snippet.html' %} -->
+{% block content %}
+
+{% include 'portal/search_filter_snippet.html' %}
+
+
+<div class="business-results">
+  <div class="layout">
+    <span class="search-results-counts">
+      <h4 class="mdc-typography--subtitle2 subtitle-clear-search">Showing</h4>
+      <h4 class="mdc-typography--subtitle2 subtitle-clear-search paginator-count mdc-theme--primary">{{ paginator.count }}</h4>
+      <h4 class="mdc-typography--subtitle2 subtitle-clear-search">results</h4>
+    </span>
+
+    {% for business in object_list %}
+    <a href="{{ business.get_absolute_url }}" class="link-to-business-profile">
+      <div class="mdc-card business-results-card">
+        <div class="business-category">
+          <span class="business-category-text">{{ business.status.label }}</span>
+          <span>
+            <i class="material-icons mdc-list-item__graphic mdc-theme--primary" aria-hidden="true">home</i>
+          </span>
+        </div>
+        <div class="business-name mdc-theme--primary">{{ business.registered_name }}</div>
+
+        <hr class="mdc-list-divider">
+
+        <div class="location-and-status-container">
+          <div class="business-details-location">
+              <summary class="business-summary">
+                <span class="business-details-content">
+                  <i class="material-icons mdc-list-item__graphic mdc-theme--primary" aria-hidden="true">location_on</i>
+                  <span class="header-summary">Address here</span>
```

Comment:
this isn't address, it's region - skip the muni - the portal is for one muni. So just template region in here. You know how.

Reply:
Covered